### DISCUSSION TITLE
3times qualification for teacher benefits REJECTED, do not hold out any hope for the 4th application though I submitted.

### DISCUSSION THREAD

**Discussion by FanTacyTutu** [community/community#101348] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/43268469/942858bf-b987-49fa-a769-670912ca4994)

Anyone please help~ 
If I choose the right name of our university, then I was required to enter a WRONG email address, however the address our faculty currently use is @xztu.edu.cn. So I have to type lower case version of the university name to continue. 

"Please select a different email address.
We require applicants of Xinzhou Normal University to use one of these school-issued email addresses to apply:
- xztc.edu.cn"

And the next page's map search won't work either, I can't search out the accurate position of our campus, the BingMap won't work, but still if switched to satellite view, we definitely can observe there was a new location of campus, also can search out by BaiduMap for verification. 
The final step's laptop cam verification, its too blurry barely can see. Besides, the faculty card we use don't have an expired date like other countries. The card is self-explanatory, if one is not currently enrolled as a faculty member, one can't hold a card.
In short, I don't know what else to do to verify. Please help me out, thx.
